lib/modules/ens/index.js

```diff
@@ -361,6 +363,15 @@ class ENS {
     this.embark.addProviderInit('names', code, shouldInit);
     this.embark.addConsoleProviderInit('names', code, shouldInit);
   }
+
+  ensResolve(name, cb) {
+    const self = this;
+    if(!self.configured) {
+      return cb('ENS not configured');
+    }
+    const hashedName = namehash.hash(name);
+    self.resolverContract.methods.addr(hashedName).call(cb);
```

Comment:
Actually, you,re right, I forgot that you need to get the right Resolver first. You can still use `this.registryContract`, but you need to get the Resolver's address, my bad.